src/adservice/src/main/java/hipstershop/AdService.java

```diff
@@ -213,21 +213,21 @@ private void blockUntilShutdown() throws InterruptedException {
         .build();
   }
 
-  private static void initStackdriver() {
-    logger.info("Initialize Stackdriver");
-
+  private static void initStats() {
+    String statsEnv = System.getenv("STATS");
+    if (statsEnv == null) {
+      statsEnv = "true";
```

Comment:
this is kinda nasty. especially because 1) it's a string 2) the yaml has capitalized values like "False".